src/hook.ts

```diff
@@ -0,0 +1,23 @@
+import { getNamespace } from 'cls-hooked'
+import { getHookInContext, NAMESPACE_NAME } from './common'
+
+export const getTransactionalContextHook = () => {
```

Comment:
I think this function should not be exported. It should be kept private to the `runOnTran**` functions